lib/ansible/plugins/action/assert.py

```diff
@@ -38,8 +38,11 @@ def run(self, tmp=None, task_vars=None):
             raise AnsibleError('conditional required in "that" string')
 
         msg = None
+        success_msg = None
         if 'msg' in self._task.args:
             msg = self._task.args['msg']
+        if 'success_msg' in self._task.args:
+            success_msg = self._task.args['success_msg']
```

Comment:
You should ensure it is a string (see string_types)

Reply:
Thanks @bcoca , done.  Wondering if `that` input needs checking too.

Reply:
all parameters should, that is something we are revising now, but I'm not going to hold you up based on existing flaws, just trying to not add to the problem at this point.